src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
```

Comment:
The `.header` block should use variables for colors and spacing (padding, margin) to ensure consistency across the project. Check if there are predefined variables in the project you can use for `max-width`, `padding`, and `margin`.